app/views/wizards/claims/add_mentor_wizard/_check_your_answers_step.html.erb

```diff
@@ -0,0 +1,54 @@
+<% content_for :page_title, t(".page_title", contextual_text:) %>
```

Comment:
Same here with the school name needing to be included in the page title, if it's a support user.